internal/etc/magicbuilder/pgp.go

```diff
@@ -8,11 +8,29 @@ import (
 	"github.com/quan-to/chevron/internal/keymagic"
 	"github.com/quan-to/chevron/internal/vaultManager"
 	"github.com/quan-to/chevron/pkg/interfaces"
+	"github.com/quan-to/chevron/pkg/models"
 	"github.com/quan-to/slog"
```

Comment:
Same comment for [imports](https://github.com/golang/go/wiki/CodeReviewComments#imports)